qrenderdoc/Windows/MainWindow.cpp

```diff
@@ -1408,8 +1408,9 @@ void MainWindow::on_action_Attach_to_Running_Instance_triggered()
 
 void MainWindow::on_action_Manage_Remote_Servers_triggered()
 {
-  // the manager deletes itself when all lookups terminate
-  RDDialog::show(new RemoteManager(m_Ctx, this));
+  RemoteManager *dialog = new RemoteManager(m_Ctx, this);
+  RDDialog::show(dialog);
+  delete dialog;
```

Comment:
It still crashed but this time during the while(dialog->isVisible()) check:

1   QWidget::testAttribute                                         qwidget.h          857
2   QWidget::isVisible                                             qwidget.h          819
3   RDDialog::show                                                 QRDUtils.cpp       356
4   MainWindow::on_action_Manage_Remote_Servers_triggered          MainWindow.cpp     1412
5   MainWindow::on_action_Attach_to_Running_Instance_triggered     MainWindow.cpp     1406
6   MainWindow::qt_static_metacall                                 moc_MainWindow.cpp 200

Self-deleting just doesnt seem safe when the calling code is still doing UI logic on the QDialog.